### DISCUSSION TITLE
posts/misra-c-2012/

### DISCUSSION THREAD

**Discussion by giscus** [ProphetHJK/prophethjk.github.io#23] [General]
# posts/misra-c-2012/

术语中英表

https://hjk.life/posts/misra-c-2012/

**Comment by 0x0059** [comment]
看到博主的头像就在想是不是去月球里面的纸兔子，看了一下关于页面，还有一首去月球的音乐。😂

**Comment by evil0knight** [comment]
nb的解读,可以仔细研读一下
